views/tx.tmpl

```diff
@@ -143,18 +141,18 @@
                 </thead>
                 <tbody>
                     {{range .Vout}}
-                    {{ if ne .Value 0.0}}
+                    {{ if ne .Amount 0.0}}
                     <tr>
-                        <td class="break-word mono">
-                            {{range .ScriptPubKeyDecoded.Addresses}}
-                                <a href="/explorer/address/{{.}}">{{.}}</a>
+                        <td>
+                            {{range .Addresses}}
+                                <a class="break-word mono" href="/explorer/address/{{.}}">{{.}}</a><br>
```

Comment:
Just want to make sure losing the `class="break-word mono" ` part was intentional since the rebase was pretty epic.  a `break-word` also disappeared way above too so I'm guessing this wasn't a mistake, but I want to be sure.